### PR TITLE
Lewis2016

### LINKED ISSUE #190: Lewist-2016-4??
http://www.sciencedirect.com/science/article/pii/S0010027716300919

Paper contains interesting norm data on word frequency and complexity based on experiments for some 490 words. Not all can be mapped, but they may be interesting as metadata.

### LINKED ISSUE #369: Lewis-2016-500
This is interesting norm data: https://github.com/mllewis/RC/tree/master/data/norms

Thanks to @blasid for pointing this out. Don't forget to put him on the CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by Schweikhard**
Adding the data set from Lewis2016 that was already prepared and mapped in the norm data project.

I'm not sure about the tags, is "ranked" correct?
I added the conceptlist and a json, and updated the references.bib and the conceptlists.tsv. Anything missing or wrong? Been a while since I last added a new concept list.

fixes #190
fixes #369

### CODE REVIEW

**Comment by Schweikhard** [comment]
Hm, on my machine concepticon test says everything is fine?

**Comment by Schweikhard** [comment]
Ah, it's complaining about a reference that I cite in the note field.
And I don't know what's wrong with the metadata - is this about the json-file?

**Comment by AnnikaTjuka** [comment]
Did you update pyconcepticon? 
And your metadata.json looks different from the one I used for #686

**Comment by AnnikaTjuka** [comment]
Also there are some extra quotations marks in the description column in the conceptlists.tsv

**Comment by Schweikhard** [comment]
My installation of pyconcepticon is quite recent.
And I merely adapted some other existing json.
But I did give the json the wrong name, that was the issue with that.

How do I handle references that are not references for the data set itself but for the explanation of the data set?

**Comment by Schweikhard** [comment]
> Also there are some extra quotations marks in the description column in the conceptlists.tsv

I don't know for sure but I think I heard that would be actually the standard style for tsv-files.

**Comment by AnnikaTjuka** [comment]
I've just tested your branch on my computer and it seems that the problem is the additional reference (Wilson 1988) that you added in the reference file. 

If I understand correctly, this is something we can only do in our norm data repository.

**Comment by LinguList** [comment]
@Schweikhard, if you add a reference (good we are doing this now), you have to mention it in your conceptlists.tsv description. So you write: `This data makes use of the data in [Wilson 1998](:bib:Wilson1998), to change the world's fate.`

**Comment by LinguList** [comment]
This is useful, since it increases awareness and also makes sure we at least mention the additional data in a given concept list actively.

**Comment by Schweikhard** [comment]
Ah I see - fixed that.
Also removed the additional quotation marks, or should they stay?

**Comment by LinguList** [comment]
They are useful for display in github, as you'll see. But only if they are in ALL the file.

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -278,3 +278,4 @@ Joo-2019-100	Joo, Ian	2019		100	ranked, basic	English	Global		Joo2019		This list
...
 Lynott-2013-400	Lynott, Dermot and Connell, Louise	2013		400	ranked	English	English	https://link.springer.com/article/10.3758/s13428-012-0267-0	Lynott2013		This list of 400 items was used to elicit modality exclusivity norms of selected noun concepts, for which participants provided perceptual strength ratings across five sensory modalities (i.e., hearing, taste, touch, smell, and vision).		
+Lewis-2016-499	Lewis, Molly L. and Frank, Michael C.	2016		499	ranked	English	English	http://www.sciencedirect.com/science/article/pii/S0010027716300919	Lewis2016		The dataset consists of "499 English words from the MRC Psycholinguistic Database [Wilson 1988](:bib:Wilson1988) that were broadly distributed in their length and were relatively high frequency." (Lewis and Frank 2016: 191) "with bootstrapped 95% CIs" (https://github.com/mllewis/RC/blob/master/data/norms/README%20-%20norms.md)
```
Comment (LinguList) [inline concepticondata/conceptlists.tsv]: URLs should be added in markdown, like `see [README norms.md](https://...)` .

Quotes can be replaced by `&quot;` The HTML counterpart, which will be displayed in CLLD as `"`.

Comment (Schweikhard) [reply concepticondata/conceptlists.tsv]: Done, thanks!

File: concepticondata/conceptlists.tsv [hunk 2]
```diff
@@ -278,3 +278,4 @@ Joo-2019-100	Joo, Ian	2019		100	ranked, basic	English	Global		Joo2019		This list
...
 Lynott-2013-400	Lynott, Dermot and Connell, Louise	2013		400	ranked	English	English	https://link.springer.com/article/10.3758/s13428-012-0267-0	Lynott2013		This list of 400 items was used to elicit modality exclusivity norms of selected noun concepts, for which participants provided perceptual strength ratings across five sensory modalities (i.e., hearing, taste, touch, smell, and vision).		
+Lewis-2016-499	Lewis, Molly L. and Frank, Michael C.	2016		499	ranked	English	English	http://www.sciencedirect.com/science/article/pii/S0010027716300919	Lewis2016		The dataset consists of &quot;499 English words from the MRC Psycholinguistic Database [Wilson 1988](:bib:Wilson1988) that were broadly distributed in their length and were relatively high frequency.&quot; (Lewis and Frank 2016: 191) &quot;with bootstrapped 95% CIs&quot; see [README norms.md](https://github.com/mllewis/RC/blob/master/data/norms/README%20-%20norms.md)
```
Comment (LinguList) [inline concepticondata/conceptlists.tsv]: sorry for being picky here, but a dot in the end of the description is missing.

Comment (Schweikhard) [reply concepticondata/conceptlists.tsv]: Done ^^

**Comment by LinguList** [comment]
okay, merge if the tests are okay, please.

**Comment by LinguList** [comment]
but resolve conflicts first, please.

**Pull request merged**